File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/dto/UserCaseGroupDto.java
package org.javamaster.invocationlab.admin.model.dto;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

/**
 * 用例分组
 *
 * @author yudong
 */
@Data
public class UserCaseGroupDto {
    String value;
    String label;
    List<UserCaseGroupDto> children = new ArrayList<>();
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/dto/UserCaseGroupDto.kt
package org.javamaster.invocationlab.admin.model.dto

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * 用例分组
 *
 * @author yudong
 */

@AllOpen
class UserCaseGroupDto {
    var value: String? = null
    var label: String? = null
    var children: List<UserCaseGroupDto> = ArrayList()
}
